x/ecocredit/core/genesis_test.go

```diff
@@ -545,3 +400,18 @@ func TestGenesisValidate(t *testing.T) {
 		})
 	}
 }
+
+// setupStateAndExportJSON setups up state as defined in the setupFunc function and then exports the ORM data as JSON.
+func setupStateAndExportJSON(t *testing.T, setupFunc func(ctx context.Context, ss api.StateStore)) json.RawMessage {
+	ormCtx := ormtable.WrapContextDefault(ormtest.NewMemoryBackend())
+	modDB, err := ormdb.NewModuleDB(&ecocredit.ModuleSchema, ormdb.ModuleDBOptions{})
+	require.NoError(t, err)
+	ss, err := api.NewStateStore(modDB)
+	require.NoError(t, err)
+	setupFunc(ormCtx, ss)
+	target := ormjson.NewRawMessageTarget()
+	require.NoError(t, modDB.ExportJSON(ormCtx, target))
+	jsn, err := target.JSON()
+	require.NoError(t, err)
+	return jsn
+}
```

Comment:
I don't think this is necessary unless we are planning on reusing it but ok to leave as is.